controllers/pointController.js

```diff
@@ -0,0 +1,25 @@
+require("../models/Photo");
+const Point = require("../models/Point");
+const { ERROR_MESSAGE } = require("../constants");
+
+const getPhotos = async (req, res, next) => {
+  const { id } = req.params;
+
+  try {
+    const { photos } = await Point.findById({ _id: id })
```

Comment:
이 부분은 아마

```suggestion
    const { photos } = await Point.findById(id)
```

이렇게만 작성하셔도 잘 불러와질거예요!!

Reply:
findOne 을 사용해 특정 필드값을 가져오는지도 확인해보았었는데 메소드를 바꾸면서 이 부분을 변경하지 않았네요ㅜ 수정하겠습니다~!